wandb/integration/tensorboard/monkeypatch.py

```diff
@@ -17,14 +18,20 @@ def patch(save=None, tensorboardX=None, pytorch=None):
 
     wandb.util.get_module("tensorboard", required="Please install tensorboard package")
     c_writer = wandb.util.get_module(TENSORBOARD_C_MODULE)
-    tb_writer = wandb.util.get_module(TENSORBOARD_PYTORCH_MODULE)
+    tb_writer = wandb.util.get_module(TENSORBOARD_WRITER_MODULE)
+    pt_writer = wandb.util.get_module(TENSORBOARD_PYTORCH_MODULE)
 
     if c_writer:
```

Comment:
I thought the Tahoe person implied that this was present, but that she needed the _patch_nontensorflow. Is that possible?

Reply:
it was implied, but i wont do that change until i have a sample and validate the impact of doing both c_writer and tb_writer.  This fixes the sample code we do have.

Reply:
fixed